src/redis-benchmark.c

```diff
@@ -1408,6 +1502,13 @@ int parseOptions(int argc, char **argv) {
         if (!strcmp(argv[i],"-c")) {
             if (lastarg) goto invalid;
             config.numclients = atoi(argv[++i]);
+        } else if (!strcmp(argv[i], "--dist")) {
```

Comment:
I don't think it's necessary to add this argument, the granularity is too fine. We just need to make sure the commands are sent smoothly.

Reply:
it's true this argument is unnecessary at present.